instructure/canvas-ios
# Add pronouns to Submissions and Speed Grader

refs: MBL-13763
affects: teacher
release note: Added support for user pronouns

Test plan:
* Pronouns should show up in the following places:
  - Submissions list
  - Speed Grader header
  - Speed Grader comments
* narmstrong : t
* CS 1400 > Assignments > File Upload > Submissions
* Pronoun for Student1 should show on submissions list
* Tap Student1
* Speed Grader header should display pronouns
* View Comments
* Student1 name in comments should include pronouns
